src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+
+  &__star {
+    background: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &--2 .stars__star:nth-child(-n + 4) {
```

Comment:
add styles for all possible cases to make it flexible (just copy styles from stars task